src/styles/utils/_media.scss

```diff
@@ -0,0 +1,17 @@
+@media screen and (min-width: 488px) {
+  .hero {
+    grid-template-columns: repeat(2, 200px);
+  }
+}
+
+@media screen and (min-width: 768px) {
+  .hero {
+    grid-template-columns: repeat(3, 200px);
+  }
+}
+
+@media screen and (min-width: 1024px) {
+  .hero {
+    grid-template-columns: repeat(4, 200px);
+  }
+}
```

Comment:
This should be right next to the `.hero` styles, because these are connected. There is no need to create a file for media queries only in this task